python/ray/tune/utils/log.py

```diff
@@ -40,6 +55,45 @@ def has_verbosity(level: Union[int, Verbosity]) -> bool:
     return verbosity_level >= log_level
 
 
+# TODO: find a file under air instead of tune to host this.
+#################################
+#         NEW FLOW              #
+#################################
+
+
+#  Note: also need to ship this env var to remote actor.
+def has_verbosity_new(level: int) -> bool:
+    if "AIR_VERBOSITY" not in os.environ:
```

Comment:
the reason we introduce a new env var is because we don't want to reuse the existing verbosity config?